src/main/java/mate/academy/spring/model/dto/request/UserRequestDto.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.spring.model.dto.request;
+
+import jakarta.validation.constraints.NotNull;
+import mate.academy.spring.validation.Email;
+
+public class UserRequestDto {
+    @Email
+    @NotNull
+    private String email;
```

Comment:
```suggestion
    @Email
    private String email;
```
you are checking if email is null in validator